src/embed/types.rs

```diff
@@ -90,6 +92,87 @@ pub struct GetResponse {
     pub has: bool, // Second to avoid trailing comma if value == None.
 }
 
+#[derive(DeJson)]
+pub struct ScanKey {
+    value: String,
```

Comment:
We use `String` for keys and values at the bindings layer but `Vec<u8>` at key/value layer. We should figure this out someday.

Reply:
heh